test/pages/SignupPage.ts

```diff
@@ -0,0 +1,35 @@
+import { Locator, Page } from "@playwright/test";
+
+export class SignupPage{
+    page:Page
+    signupLinkLocator:Locator
+    emailFieldLocator:Locator
+    screenNameFeldLocator:Locator
+    passwordFieldLocator:Locator
+    signupButton:Locator
+
+    constructor({page}:{page:Page}){
+        this.page = page
+        this.signupLinkLocator = page.getByRole('link',{name:"Sign Up"})
```

Comment:
Please fix these kind of spacing also
```suggestion
        this.signupLinkLocator = page.getByRole('link', {name:"Sign Up"})
```